components/Filters/helpers.ts

```diff
@@ -32,3 +28,45 @@ export const FILTER_PLATFORMS = [
     title: 'Windows',
   },
 ];
+
+export const FILTER_REQUIRES_MAIN_SEARCH = [
+  {
+    param: 'isMaintained',
+    title: 'Maintained',
+  },
+  {
+    param: 'isPopular',
+    title: 'Popular',
+  },
+  {
+    param: 'isRecommended',
+    title: 'Recommended',
+  },
+];
+
+export const FILTER_STATUS = [
```

Comment:
Since we already alter the code, can we also reorder those filters?

Suggested order:
* new arch
* example
* images
* types
* expo go
* updated

Reply:
Done